config/peerpods/podvm/lib.sh

```diff
@@ -229,6 +229,11 @@ function prepare_source_code() {
         create_overlay_mount_unit
```

Comment:
I think `adapt_cloud_init` can be called here ..

Reply:
not an option ATM as this should be disabled for libvirt

Reply:
How about something like the following ?
[[ "$CLOUD_PROVIDER" != "libvirt" ]] && adapt_cloud_init

Reply:
If you want the ability to enable/disable the cloud-init modules explicitly then it makes sense to have it separately..
Can you please change the config param to CUSTOM_CLOUD_INIT_MODULES and the function name to set_custom_cloud_init_modules